voorhoede/tudelft-spacefinder
# docs: data sources, data flow and status

# Changes

- adds documentation on data sources and data flow in the project readme.
- adds diagram to visualise data sources and data flow into the application.

# Associated issue

Resolves https://trello.com/c/Sm4FJtxq/40-document-data-sources-flow

# How to test

- Navigate to ["Files changed" in this PR](https://github.com/voorhoede/tudelft-spacefinder/pull/160/files).
- For `README.md` select ["View file"](https://github.com/voorhoede/tudelft-spacefinder/blob/bdc05c14eeaf2f27a2bda4ac7dbd4cb80c476c28/readme.md#data).
- Verify the "Data" section correctly describes the data sources and data flow.
- Verify the "Data" section diagram correctly visualises the data flow into the application.

# Checklist
